src/components/quantityLegend.js

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+
+const QuantityLegend = ({
+  text,
+  boxSize,
+  imgSrc,
+  background,
+  quantity,
+  isPercent,
+  invertedBoldText,
+  wrapStyle,
+  boxStyle,
+  descTextStyle,
+}) => {
+  const quantityDisplayText = () => (isPercent ? `${quantity}%` : `${quantity}`)
+
+  return (
+    <div
+      css={{
+        marginRight: "1rem",
+        display: "flex",
+        alignItems: "baseline",
+        fontSize: "unset",
+        whiteSpace: "nowrap",
+        ...wrapStyle,
+      }}
+    >
+      {imgSrc ? (
+        <img
+          src={imgSrc}
+          width={boxSize}
+          height={boxSize}
+          style={{
+            marginRight: "4px",
+          }}
+          alt={text}
+        />
+      ) : (
+        <div
+          css={{
+            display: "inline-box",
+            boxSizing: "border-box",
+            border: `1px solid ${background ?? "var(--cl-black)"}`,
```

Comment:
This one works fine, no need to change it. But I want to point out that CSS var support fallback value with the 2nd argument :) https://developer.mozilla.org/en-US/docs/Web/CSS/var